### ISSUE TITLE
[Content organization] Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page. (02.04.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#50951]
#### VFS team name
DGIB - My Education Benefits

#### VFS product name
Authenticated 22-1990e Transfer of Entitlement (TOE) Form Updates

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Your page doesn't follow heading hierarchy. Headings aren't properly nested, and headings and/or labels aren't relevant to the page.

### Issue Details
In Step 2, there are two instances of H3 headings being immediately followed by another H3 heading: 

- When the test user doesn't have a sponsor on file and receives a "We don't have any sponsor information" warning with an H3 immediately following the "Enter your sponsor's information" H3. 
- When the test user needs to verify high school education, a "We need additional information" info alert with an H3 immedediately follows a "Verify your high school education" H3.

Two consecutive H3s with no content between them can be confusing to screen reader users, who may be expecting them to be separate sections of the page each with their own content. They might assume there's an empty section of content or that something is broken.

### VA.gov Experience Standard
[Category Number 02, Issue Number 04](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A
WCAG SC 2.4.6_AA

### Platform Recommendation
If I'm understanding the intent of the alerts correctly, I think both of these work if the alert is moved up before the other H3. The content flow would make a little more sense to screen reader users (H3, related content, another H3, more content). And the content itself might make more sense this way -- I believe @CherylEvans has another feedback ticket related to this.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by btruongafs** [comment]
Issue resolved and deployed to staging.
